Rettelse PR (#62)
MrLongbottom/P9-Rapport: sections/Dataset.tex

\section{Dataset}\label{sec:dataset}
Nordjyske is a media group maintaining many local newspapers, radios, and websites in the North Jutland region of Denmark.
All articles produced by Nordjyske are saved in a nonpublic database.
Each article is represented with a \texttt{pdf} file and an \texttt{xml} file, as well as one \texttt{png} file for each image in the article.

> The dataset we are using is only a subset of all articles contained in the database.

Author: jeg er i tvivl om der er nok til både at have noget i main paper og appendix, og føler det er lidt sært overhovedet ikke at nævne dataset i main paper. Men du er velkommen til at give det et forsøg med en PR på et tidspunkt and prove me wrong

> We choose to take all articles written from 2017-2019, due to the file structures being more consistent after 2016.

Author: ```suggestion
We choose to take all articles published in 2017, due to the file structures being more consistent after 2016.
```

> We also wanted to include 2020, however, there was a period where no articles were uploaded to the database around the time of the Covid-19 outbreak, so a lot of articles seem to be missing from this year, unfortunately.

Author: ```suggestion
```

Currently we extract only the ID, body text, and headline from the xml files, though they do contain more information, such as authors, references to images, article type, date and time of publishing, etc.

> This information could become relevant with the use of various \gls{lda} modifications.

Author: ```suggestion
This information could become relevant with the use of various \gls{lda} modifications, however this is beyond the scope of this paper.
```
